### ISSUE TITLE
Implement comments in Entry Information Page

### ISSUE THREAD

**Issue by richpomfret** [FreeUKGen/FreeBMD2#337]
As above.

**Comment by Vino-S** [comment]
Ready for testing in test3

**Comment by richpomfret** [comment]
Ability to view comments on entry information page.

**Comment by Vino-S** [comment]
Example of a Record with a comment:

First Name: John
Surname: Callum
Event Type: Birth
Event Date: September 1940
Volume: 9a
Page: 898

**Comment by PhilipClarke712** [comment]
Hi @Vino-S 

Out of interest I searched for that record on test3, it works as expected and has it's comments. 

I also searched for the names given in issue #308: marriage(s) of William R Betts (1950) - wife has two surnames; marriage of Richard George Alcock (1881) and birth of Edith Alcock (1891) and those are also working as expected in respect of their comments.

I've not checked any Late Entries, yet.

I'll try and find some comments of the form "COMMENT(4) entry reads Smith or JONES for mother's name & handwritten amendment of crossed out Page No. 1950", but these may not be very common in records  older than the 1970s. [That  obviously means there are four consecutive records with the same comment, rather than two consecutive records).

Presumably, **Postems** should also appear in that box, I'll try and find some Postems in that data set, but they are apparent quite rare: Barry made comments about that on a different issue.

This is going a bit off-issue: but none of those records have any scan references, so I can't check what is given against a scan.
